tfutils/base.py

```diff
@@ -742,15 +749,15 @@ def train_from_params(save_params,
 def get_valid_targets_dict(validation_params, 
                            model_func, model_kwargs,
                            default_queue_params,
-                           cfg_final):
+                           cfg_final,
+                           original_seed):
     """NB: this function may modify validation_params"""
     valid_targets_dict = OrderedDict()
     queues = []
-
     for vtarg in validation_params:
         if 'targets' not in validation_params[vtarg]:
             validation_params[vtarg]['targets'] = default_loss_params()
-        if 'func' not in validation_params[vtarg]['targets']:
+        if 'func' not in validatfion_params[vtarg]['targets']:
```

Comment:
Fixed -- there's something wrong with agent1 making it next to impossible to run the tests -- things are moving so slow.